falcon/testing/__init__.py

```diff
@@ -115,6 +115,50 @@ def test_get_message(client):
 from falcon.testing.srmock import StartResponseMock
 from falcon.testing.test_case import TestCase
 
+__all__ = [
+    # client
+    'ASGIConductor',
+    'Cookie',
+    'Result',
+    'ResultBodyStream',
+    'simulate_delete',
+    'simulate_get',
+    'simulate_head',
+    'simulate_options',
+    'simulate_patch',
+    'simulate_post',
+    'simulate_put',
+    'simulate_request',
+    'StreamedResult',
+    'TestClient',
+    # helpers
+    'ASGILifespanEventEmitter',
+    'ASGIRequestEventEmitter',
+    'ASGIResponseEventCollector',
+    'ASGIWebSocketSimulator',
+    'closed_wsgi_iterable',
+    'create_asgi_req',
+    'create_environ',
+    'create_req',
+    'create_scope',
+    'create_scofrom_headers',
+    'get_unused_port',
+    'rand_string',
+    'redirected',
+    # resource
+    'capture_responder_args',
+    'capture_responder_args_async',
+    'set_resp_defaults',
+    'set_resp_defaults_async',
+    'SimpleTestResource',
+    'SimpleTestResourceAsync',
+    # srmock
+    'StartResponseMock',
+    # test_case
+    'TestCase',
+]
+
+
 # NOTE(kgriffs): Alias for backwards-compatibility with Falcon 0.2
 # TODO: remove in falcon 4
```

Comment:
@vytas7 we forgot I guess :/

Reply:
```suggestion
# TODO: remove in falcon 5
```

Reply:
would you like this as part of this PR or is it better to have it in another change? (it is somewhat unrelated)

(Also, a trick we use in our code is to have markers on scheduled TODO items, so this would become something like `TODO(falcon5): remove in falcon 5` so it's an easy grep when the time is right to release falcon 5)

Reply:
No, that would be wrong according to our conventions :slightly_smiling_face: 
It should be just `TODO` or `TODO(nickname)`.

Reply:
Probably it would be nice to plop a `@deprecation.deprecated(...)` decorator on top of this alias, and add a test checking the warning.
But I would prefer it as a separate improvement, nice if you could create a PR @jap :slightly_smiling_face:

Reply:
I created an issue for this: #2389.